src/server/index.js

```diff
@@ -520,6 +523,78 @@ module.exports = async function startServer(options) {
     ctx.body = { url, sceneId: scene_id };
   });
 
+  const nodePlatformToAssetPlatform = {
+    win32: "win",
+    darwin: "macos",
+    linux: "linux"
+  };
+  function getDownloadUrlForCurrentPlatform(assets) {
+    const assetPlatform = nodePlatformToAssetPlatform[process.platform];
+    return assets.find(asset => asset.name.includes(assetPlatform)).downloadUrl;
+  }
+
+  const updateInfoTimeout = 2000;
+
+  async function getLatestRelease() {
+    const token = "de8cbfb4cc0281c7b731c891df431016c29b0ace";
+    const result = await fetch("https://api.github.com/graphql", {
+      timeout: updateInfoTimeout,
+      method: "POST",
+      headers: { authorization: `bearer ${token}` },
+      body: JSON.stringify({
+        query: `
+          {
+            repository(owner: "mozillareality", name: "spoke") {
+              releases(last: 1, isPrerelease: false, isDraft: false) {
+                nodes {
+                  tag { name }
+                  releaseAssets(last: 3) {
+                    nodes { name, downloadUrl }
+                  }
+                }
+              }
+            }
+          }
+        `
+      })
+    }).then(tryGetJson);
+
+    if (!result || !result.data) return;
+
+    const release = result.data.repository.releases.nodes[0];
+
+    return {
+      version: release.tag.name,
+      downloadUrl: getDownloadUrlForCurrentPlatform(release.releaseAssets.nodes)
+    };
+  }
+
+  router.get("/api/update_info", koaBody(), async ctx => {
+    try {
+      // This endpoint doesn't exist yet but we query it for future use.
+      const configEndpoint = `https://${reticulumServer}/api/v1/config`;
```

Comment:
to be a bit pedantic, this should probably be `/api/v1/config/spoke`

Reply:
actually this is non-pedantic since i guess this URL actually matters :)

Reply:
`/api/v1/configs/spoke`?